api/bases/dataplane.openstack.org_openstackdataplanedeployments.yaml

```diff
@@ -0,0 +1,87 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.11.1
+  creationTimestamp: null
+  name: openstackdataplanedeployments.dataplane.openstack.org
+spec:
+  group: dataplane.openstack.org
+  names:
+    kind: OpenStackDataPlaneDeployment
+    listKind: OpenStackDataPlaneDeploymentList
+    plural: openstackdataplanedeployments
+    shortNames:
+    - osdpd
+    - osdpdeployment
+    - osdpdeployments
+    singular: openstackdataplanedeployment
+  scope: Namespaced
+  versions:
+  - additionalPrinterColumns:
+    - description: Status
+      jsonPath: .status.conditions[0].status
+      name: Status
+      type: string
+    - description: Message
+      jsonPath: .status.conditions[0].message
+      name: Message
+      type: string
+    name: v1beta1
+    schema:
+      openAPIV3Schema:
+        properties:
+          apiVersion:
+            type: string
+          kind:
+            type: string
+          metadata:
+            type: object
+          spec:
+            properties:
+              ansibleLimit:
+                type: string
+              ansibleSkipTags:
+                type: string
+              ansibleTags:
+                type: string
+              nodeSets:
```

Comment:
Nice, this looks like the most ansible-native representation of an inventory, where nodeSets represent ansible groups - without goinng into the raw inventory yaml representation